cmd/bootstrap/runner.go

```diff
@@ -545,6 +528,16 @@ func (r *runner) installOperator(ctx context.Context, helmClient helmclient.Inte
 
 		r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("installing %#q", name))
 
+		// Set operator values so chart-operator DNS settings are correct.
```

Comment:
I don't see any DNS related values from `operatorValuesYAML`. Maybe can you point it out?

Reply:
We check if the values has the `Installation` key. https://github.com/giantswarm/chart-operator/blob/13df537924a1ed71b195d0981fcbae37bad93445/helm/chart-operator/templates/deployment.yaml#L45-L72

I'll update the comment so say these are the control plane values. As otherwise it's confusing.

Reply:
@tomahawk28 Done. PTAL